build.gradle

```diff
@@ -105,6 +105,7 @@ subprojects { Project subproject ->
     tasks.withType(JavaCompile).configureEach {
        options.compilerArgs.add("-Amicronaut.processing.group=$project.group")
        options.compilerArgs.add("-Amicronaut.processing.module=micronaut-$project.name")
+       options.compilerArgs.add("-Amicronaut.omit.confprop.injectpoints=true")
```

Comment:
Rename to `micronaut.processing.omit.confprop.injectpoints` and I believe you will need to add it to https://github.com/micronaut-projects/micronaut-core/blob/6af280483f86f1c42ce8b1c010aa1ddd90176b5e/inject-java/src/main/java/io/micronaut/annotation/processing/BeanDefinitionInjectProcessor.java#L77

Reply:
Done